src/Mod/TechDraw/TDTest/TDPyTest.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# basic test script for TechDraw Py functions migrated from Drawing
+
+from __future__ import print_function
+
+import FreeCAD
+import Part
+import Measure
```

Comment:
Doesn't have anything to do with the moving either, but some unused modules could be removed.

```suggestion
# import Part
```